source/FreeRTOS_Sockets.c

```diff
@@ -5133,6 +5137,9 @@ void * pvSocketGetSocketID( const ConstSocket_t xSocket )
  *        and return the value -pdFREERTOS_ERRNO_EINTR ( -4 ).
  *
  * @param[in] xSocket: The socket that will be signalled.
+ * 
+ * @return If xSocket is an invalid socket then -pdFREERTOS_ERRNO_EINVAL 
+ *         is returned. Else 0 is returned.
```

Comment:
This seems a bit incomplete. Here, the `-pdFREERTOS_ERRNO_EINVAL` is returned whenever there is an invalid input. that may be socket being invalid or the event group not being created and/or the socket set being NULL.

We would need to modify this comment to reflect that.

Thanks